addons/l10n_cl/models/res_partner.py

```diff
@@ -21,3 +22,32 @@ class ResPartner(models.Model):
              '2 - Fees Receipt Issuer (Applies to suppliers who issue fees receipt)\n'
              '3 - End consumer (only receipts)\n'
              '4 - Foreigner')
+
+    def _format_vat_cl(self, values):
+        identification_types = [self.env.ref('l10n_latam_base.it_vat').id, self.env.ref('l10n_cl.it_RUT').id,
+                                self.env.ref('l10n_cl.it_RUN').id]
+        country_id = self.env.ref('base.cl').id
+        if values.get('country_id') and values.get('country_id') == country_id and \
+                values.get('l10n_latam_identification_type_id') and values.get('l10n_latam_identification_type_id') in \
+                identification_types and values.get('vat'):
```

Comment:
goed do values.get('l10n_latam_ident... in ... immediately without the and

Reply:
changed the logic to use the formatting when partner country is Chile or country is empty (assumes that needs formatting for Chile in this case also).

Reply:
removal of code done at 206476a6496f4456606066258dbd1327452033bb